src/components/ProgressViewer/ProgressViewer.tsx

```diff
@@ -86,6 +86,10 @@ export function ProgressViewer({
         } else if (fillWidth > dangerThreshold) {
             status = inverseColorize ? 'good' : 'danger';
         }
+        if (!isNumeric(capacity)) {
+            status = 'good';
```

Comment:
Please, do not set status here. In nodes memory case it will be good anyway, but this will break the case with `inverseColorize=true`